src/lib/broadcast.jl

```diff
@@ -76,6 +76,11 @@ Numeric{T<:Number} = Union{T,AbstractArray{<:T}}
   res, Δ -> (nothing, unbroadcast(x, Δ ./ y), unbroadcast(y, -Δ .* res ./ y))
 end
 
+@adjoint function broadcasted(::typeof(Base.literal_pow), ::typeof(Main.:^), x::Numeric, y::Val{p}) where p
```

Comment:
```suggestion
@adjoint function broadcasted(::typeof(Base.literal_pow), ::typeof(Main.:^), x::Numeric, ::Val{p}) where p
```

Reply:
Wait `y` is the result in the function body? 

Although I also don't see why `Main.:^` not just `^`. (In fact see my comment.)

Reply:
Good catch, that `y` type instability 😅.